Makefile.pre.in

```diff
@@ -1754,23 +1760,23 @@ Programs/python.o: $(srcdir)/Programs/python.c
 Programs/_testembed.o: $(srcdir)/Programs/_testembed.c Programs/test_frozenmain.h $(PYTHON_HEADERS)
 	$(CC) -c $(PY_CORE_CFLAGS) -o $@ $(srcdir)/Programs/_testembed.c
 
-Modules/_sre/sre.o: $(srcdir)/Modules/_sre/sre.c $(srcdir)/Modules/_sre/sre.h $(srcdir)/Modules/_sre/sre_constants.h $(srcdir)/Modules/_sre/sre_lib.h
+Modules/_sre/sre.o: $(srcdir)/Modules/_sre/sre.c $(srcdir)/Modules/_sre/sre.h $(srcdir)/Modules/_sre/sre_constants.h $(srcdir)/Modules/_sre/sre_lib.h $(srcdir)/Modules/_sre/sre_targets.h $(srcdir)/Include/internal/pycore_emscripten_trampoline.h
```

Comment:
Why `pycore_emscripten_trampoline.h`? Ditto for L1765, L1771, L1779, L1795, and all the other occurrences.

Reply:
Sorry for the mishap that caused the PR to be CLOSED, I resubmitted a new PR and addressed the issue you raised.